### PR TITLE
Use unqualified group names

### PR DESCRIPTION
**Pull request by larsks**
Previously, keycloak was providing fully qualified group names, but since
we're not using hierarchical groups this just meant that our groups were
all prefixed with `/`.

I've updated keycloak to provide unqualified group names, so this change is
necessary to make them match.

Closes cci-moc/ops-issues#492

### CODE REVIEW

**Review by naved001 — APPROVED** [review]

File: netbox/overlays/ocp-prod/config/extra.py [hunk 1]
```diff
@@ -7,6 +7,6 @@
 REMOTE_AUTH_GROUP_SEPARATOR = ","
 REMOTE_AUTH_GROUP_SYNC_ENABLED = True
 REMOTE_AUTH_HEADER = "HTTP_X_FORWARDED_PREFERRED_USERNAME"
-REMOTE_AUTH_STAFF_GROUPS = ["/netbox-admin"]
-REMOTE_AUTH_SUPERUSER_GROUPS = ["/netbox-admin"]
+REMOTE_AUTH_STAFF_GROUPS = ["netbox-admins"]
```
Comment (naved001) [inline netbox/overlays/ocp-prod/config/extra.py:10]: sorry I have been commenting after reviewing/merging. But I noticed that this is plural `-admins`. Is that change intentional?

Comment (larsks) [reply netbox/overlays/ocp-prod/config/extra.py:10]: Yes, that was intentional -- we had two groups in keycloak, and I chose that one.
